src/hooks/usePendingRequests.ts

```diff
@@ -26,23 +26,24 @@ export default function usePendingRequests() {
   >(
     "pendingRequests",
     () => {
-      return queryGetPendingRequests(contract).then((res) => {
-        if (res) {
-          return res;
-        } else {
-          return [];
-        }
-      });
+      return queryGetPendingRequests(contract)
+        .then((res) => {
+          if (res) {
+            return res;
+          } else {
+            return [];
+          }
+        })
+        .catch((err) => {
+          addError(error!.message);
+          return [] as PendingRequest[];
+        });
```

Comment:
same comment as above, this could be simplified if we assume data can be undefined in the view, and its checked there. you could even have the view be responsible for setting the defaults: `{data=[]} = usePendingRequests()`

Reply:
+1